### PR TITLE
Add parquet2sql

### PR DESCRIPTION
**Pull request by Guillemdb**
Signed-off-by: Guillem Duran <guillem@sourced.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
 # parquetry
-# parquetry
```
Comment (vmarkovtsev) [inline README.md:2]: Hehe, we should have a title in the README

File: parquet2sql/__main__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from parquet2sql.parquet2sql import get_parser, parquet2sql
+
+if __name__ == "__main__":
+    parser = get_parser()
```
Comment (vmarkovtsev) [inline parquet2sql/__main__.py:5]: Can you please move these to `def main()` - that way we can link it as an entry point in `setup.py`

File: parquet2sql/parquet2sql.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    return parser
+
+
+def parquet2sql(args):
+    log = logging.getLogger("parquet2sql")
+    df = pd.read_parquet(args.parquet)
+    engine = create_engine(args.db)
+    df.to_sql(args.table, engine)
+    log.info("File %s saved to table %s at %s" % (args.parquet, args.table, args.db))
+    return 0
```
Comment (vmarkovtsev) [inline parquet2sql/parquet2sql.py]: This is looking weird, we'd rather convert from None to 0 in `def main()`

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: parquet2sql/parquet2sql.py (hunk 3 above)
Comment (Guillemdb) [reply parquet2sql/parquet2sql.py]: Sure. I just copy pasted Konst's code. I will update it.

File: parquet2sql/__main__.py (hunk 2 above)
Comment (Guillemdb) [reply parquet2sql/__main__.py:5]: Done

File: README.md (hunk 1 above)
Comment (vmarkovtsev) [reply README.md:2]: Not really resolved, still removed :D

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+package_data = {"": ["README.md"]}
+
+setup(
+    name="parquetry",
+    description="Dump parquet files to sql",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    version=parquetry.__version__,
+    license="Proprietary",
```
Comment (vmarkovtsev) [inline setup.py]: Apache 2.0

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        "python-snappy",
+        "psycopg2-binary",
+    ],
+    package_data=package_data,
+    python_requires=">=3.5",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "License :: Propietary license",
```
Comment (vmarkovtsev) [inline setup.py]:
```suggestion
        "License :: OSI Approved :: Apache Software License",
```

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        "Operating System :: POSIX",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```
Comment (vmarkovtsev) [inline setup.py:45]:
```suggestion
        "Programming Language :: Python :: 3.7",
        "Programming Language :: Python :: 3.8",
```

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

**Comment by Guillemdb** [comment]
What should I do with the README? Doesn't already look like it should? https://github.com/Guillemdb/parquetry?organization=Guillemdb&organization=Guillemdb

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**